presentation/frontend/src/test/favorite-properties/FavoritePropertiesManager.test.ts

```diff
@@ -25,14 +25,12 @@ describe("FavoritePropertiesManager", () => {
   let nestedContentField: NestedContentField;
   const storageMock = moq.Mock.ofType<IFavoritePropertiesStorage>();
 
-  // SWB
-  let projectId: string;
+  let itwinId: string;

```

Comment:
Breaking convention with itwinId and the lowercase T

Reply:
Correcting spelling for consistency is beyond the scope of this PR and will be done in the future.